### DISCUSSION TITLE
VR

### DISCUSSION THREAD

**Discussion by smmdolfin424** [community/community#69592] [Other Feature Feedback, Questions, & Ideas]
What's best most affordable VR set up?

**Comment by moayad-star** [comment]
1. Oculus Quest 2: This standalone headset offered a good balance between price and performance, with the option to connect it to a PC for more advanced VR experiences.

2. PlayStation VR: If you already owned a PlayStation console, the PS VR was a cost-effective choice. It provided a good VR experience, especially for gaming.

3. Windows Mixed Reality Headsets: These were generally more budget-friendly options for PC-based VR, with various models available from different manufacturers.

4. Mobile VR: If you have a compatible smartphone, you could explore mobile VR options like Samsung Gear VR or Google Cardboard, which were very affordable but less immersive compared to PC-based or standalone VR.
